pacstall

```diff
@@ -492,7 +490,38 @@ Helpful links:
                     echo -e "~ Rick Pacsley ~"
                     exit 0
                 fi
-                export PACKAGE=$2
+                if [[ ${2##*.} == "pacscript" ]]; then
+                    export PACKAGE="$(basename "$2" ".pacscript")" # Removes the suffix
+                    export type="install"
+                    export local="yes"
+                    PKGPATH="$(dirname "$2")"
+                    # Check if we need to cd into the directory first
+                    if [[ $PKGPATH == "." ]] && [[ ! -f "$PACKAGE".pacscript ]] && [[ -d $PACKAGE ]]; then
+                        cd "$PACKAGE"
+                    elif [[ -d $PKGPATH ]]; then
+                        cd "$PKGPATH"
+                    fi
+
+                    # Check if the file exist
+                    if [[ ! -f "$PACKAGE".pacscript ]]; then
+                        fancy_message error "$2 does not exist"
+                        shift
+                        continue
+                    fi
+                    if ! source "$STGDIR/scripts/install-local.sh"; then
+                        fancy_message error "Failed to install ${GREEN}${PACKAGE}${NC}"
+                        exit 1
+                    fi
+                    if [[ -s $LOGFILE ]]; then
+                        exit 1
+                    fi
+                    continue
```

Comment:
This is could be outside the if, since it's used by both local and remote installs